sponsorship.md

```diff
@@ -0,0 +1,46 @@
+# Pangeo Sponsorship Policy
+
+Now that Pangeo has a 501c3 fiscal sponsor, it can receive donations. Individuals or institutions who make donations to the project are **Project Sponsors**. The purpose of this document is to define the relationship between the Project and Project Sponsors.
+
+## Types of Sponsorship
+
+There are two ways to sponsor the Project:
+1. Direct financial contribution - a direct donation made via NumFocus.
+2. In-kind contribution - goods or services provided to the project in support of its activities. Examples of in-kind contributions include:
+
+   - Personnel time dedicated to project support and administration. This excludes work on the project by Steering Council members.
+   - Computers, cloud credits, or other IT-related services paid for by the sponsor on behalf of the project.
+   - Access to facilities such as office space, meeting space, or shared conference booths.
+
+  When making an in-kind donation, the sponsor should notify NumFocus of the donation and quantify the value of the in-kind donation. (Such donations are tax deductible for the sponsor.)
```

Comment:
In https://pandas.pydata.org/about/sponsors.html, we're careful with claiming that donations are tax deductible.

> For donors in the United States, your gift is tax-deductible to the extent provided by law. As with any donation, you should consult with your tax adviser about your particular tax situation.

Reply:
@TomAugspurger - I adapted this suggestion in the latest commit.

Reply:
Looks good, thanks.